gcloud/orb.yaml

```diff
@@ -166,7 +166,6 @@ commands:
         type: string
     steps:
       - run: python3 -m pip install twine keyring keyrings.google-artifactregistry-auth
-      - install
```

Comment:
perhaps, instead of doing this, can we not just update `gcloud/install` to skip install if `gcloud` is available on the `$PATH`?

Reply:
I see your point of view. After experimenting with the code, I am apprehensive of doing this for a couple reasons:

1. Changing the `install` method impacts a large number of other jobs and since the `install` job is predicated on side-effects, I'm nervous to change it
2. Every other command requires you install gcloud separately and I think it's valuable to be consistent.

I like just changing the `publish` command because it has less side effects and I know no one else depends on it. Let me know what you think.

Reply:
I think both approaches are okay. It is a matter of opinion and perhaps how hacky will it be determining if the gcloud tools have been installed correctly.

My perspective is that having multiple `gcloud/install` for the same job should be okay as this helps in using `install` say in `artifacts-package-publish` or `deploy-cloud-function` without having a hard requirement on the caller to have `gcloud` installed already (if reused as part of another job) and ensure we have a path to install `gcloud` within the task itself if needed.

Reply:
+1 for consistency :P

Reply:
Consistency is a good point though. But if we can ensure that we remove this requirement, then it simplifies the workflows for callers.

Reply:
Sorry my previous message was posted before I saw yours. If we want to encapsulate the gcloud installation within the other gcloud commands and change over where we currently do the prerequisite `gcloud/install`, I'd be fine with that as well.

Reply:
I admit, I don't have a terribly strong opinion either way. I would feel better releasing a major release if we were to  make the `install` command more robust. Given it appears there are other projects where we use the Google's alpine image where the SDK is already installed, I think it might be best to just let the caller handle the installation logic.

I really appreciate your thoughtful perspective Harsh. In light of my tinkering and experimenting, I think I'll lean towards making the caller handle the installation.

Reply:
No problem folks. Maybe we can do something like this in the future if we feel like juggling gcloud installations is becoming a pain :)